symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/config/BdkExtAppTracingFilterConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.symphony.bdk.app.spring.config;
+
+import com.symphony.bdk.app.spring.SymphonyBdkAppProperties;
+import com.symphony.bdk.app.spring.filter.TracingFilter;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Configuration of the {@link TracingFilter}, that is activated by default.
+ */
+@ConditionalOnProperty(name = "bdk-app.tracing.enabled", havingValue = "true", matchIfMissing = true)
+public class BdkExtAppTracingFilterConfig {
+
+  @Bean
+  public FilterRegistrationBean<TracingFilter> tracingFilter(SymphonyBdkAppProperties properties) {
+    final FilterRegistrationBean<TracingFilter> registrationBean = new FilterRegistrationBean<>();
```

Comment:
do we need to care about the filter's order (or expose a property to set it like in spring boot security)?
because if you use it in the logs you might have another filter executed before that one

Reply:
Fixed in https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/361/commits/80d3ba38edf9684574851a186380181539c319f9